packages/styles/components/_c.tabs.scss

```diff
@@ -0,0 +1,116 @@
+
+// .mc-tabs__nav-item--disabled.mc-tabs__nav-item--selected {
+//   color: $color-primary-01-200;
+//   border-bottom: 0.125rem solid $color-primary-01-200;
+
+//   .mc-tabs__nav-link {
+//     color: $color-primary-01-200;
+//   }
+// }
+
+/* mqp:start */
+
+.mc-tabs {
+  @include set-tag-base-style('m');
+
+  color: $color-grey-900;
+  height: auto;
+  width: 100%;
+  position: relative;
+  background: none;
+  min-height: $mu250;
+
+  &__nav {
+    margin: 0;
+    padding: 0;
+    position: absolute;
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    background: none;
+    box-shadow: none;
+    z-index: auto;
+    width: auto;
+
+    &--hidden {
+      display: flex;
+      max-width: 100%;
+      max-height: none;
+    }
+
+    &-link {
```

Comment:
~~&-link {~~ => `&__link`.
This class should not be linked to the `&-nav` selector.